evalml/pipelines/classification_pipeline.py

```diff
@@ -147,16 +147,12 @@ def score(self, X, y, objectives):
             X, y, y_predicted, y_predicted_proba, objectives
         )
 
-    def _compute_predictions(self, X, y, objectives, time_series=False):
+    def _compute_predictions(self, X, y, objectives):
```

Comment:
time series pipelines now have their own `_compute_predictions` because we need to use `_in_sample` method